### ISSUE TITLE
Add phantomjs as a browser name

### ISSUE THREAD

**Issue by just-boris** [browserslist/browserslist#406]
[PhantomJS](https://phantomjs.org) is an old browser, but still sometimes makes sense. It would be nice to be able to use it in queries.

PhantomJS uses QT Webkit as an engine. The corresponding versions are the following:

* PhantomJS 2.1 -> QT Webkit 5.5.1 -> Chrome 40
Source: https://wiki.qt.io/New_Features_in_Qt_5.5

* PhantomJS 1.9 -> QT Webkit 4.8.2 -> Safari 5 
Source: https://github.com/Financial-Times/polyfill-service/issues/208#issuecomment-65783708

**Comment by ai** [comment]
Am I right that PhantomJS is dead and we do not need to add new versions?

**Comment by just-boris** [comment]
Correct. Also `not dead` query should exclude it from the list.

**Comment by ai** [comment]
We will add it as alias (because of architecture limits), so it will not be added by `last 2 versions` and we do not need it in `not dead`

**Comment by zloirock** [comment]
@just-boris PhantomJS 5.1 does not support a big part of features which support Chrome 40. A little part of a huge listing:
![Снимок экрана 2019-08-25 в 16 02 26](https://user-images.githubusercontent.com/2213682/63647888-e9c83000-c751-11e9-8012-2e3453d864c2.png)
Seems because of Qt WebEngine uses Chromium 40, not deprecated Qt WebKit?

**Comment by just-boris** [comment]
Good point, thank you @zloirock!

I have checked it deeper.

The user agent string of PhantomJS is `Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/538.1 (KHTML, like Gecko) PhantomJS/2.1.1 Safari/538.1`

According to [this table](https://en.wikipedia.org/wiki/Safari_version_history#Safari_8), this version of Webkit, 538, was released as part of Safari 8.

@zloirock Can you please check if this matches your information about feature support?

**Comment by zloirock** [comment]
At least, those tests which pass in Safari 8.0.8 fails in PhantomJS 2.1:
```js
Math.acosh
Math.asinh
Math.atanh
Math.cbrt
Math.cosh
Math.expm1
Math.fround
Math.hypot
Math.log10
Math.log1p
Math.log2
Math.sign
Math.sinh
Math.trunc
ArrayBuffer.isView
```
I haven't Safari 8.0.0 for check it, however, some of those tests should be passed even in Safari 7.1.

**Comment by just-boris** [comment]
So, there is Safari 8.0, which is Webkit `538.35.8`
PhantomJS uses `538.1.0`, which was released before.

What's interesting is that [the version bump](https://trac.webkit.org/changeset/151792/webkit) to `538.1` was from `537.47` which corresponds to Safari 6.1. So, Safari 7 was developed and released from another feature-branch, apparently.

So, PhantomJS 2.1 is somewhere between Safari 6 and 8 in terms of features. Does anybody have any ideas how can I narrow this down?

Maybe @vitallium may help?

**Comment by ai** [comment]
We don’t need to be 100% accurate in Safari version. We just need to be sure that JS code after Babel will work in PhantomJS. So we can set Safari 6.

What about Phantom 1.9? Can we prove that it is Safari 5?

**Comment by zloirock** [comment]
I use [this mapping](https://github.com/zloirock/core-js/blob/master/packages/core-js-compat/src/mapping.js#L56-L57). At least approximately, it looks like the truth. `compat-table` contains [this mapping](https://github.com/kangax/compat-table/blob/gh-pages/environments.json#L2309-L2330). However, `phantomjs@2.1` results definitely better than `safari@6`.

**Comment by ai** [comment]
I will add only `PhantomJS 1.9` and `Phantom 2.1` aliases (case insensitive). They will not be part of any common queries like `last 2 version` or `> 1%`. Is it OK?

**Comment by just-boris** [comment]
Good for me

**Comment by ai** [comment]
Done c720da3

**Comment by ai** [comment]
Released in 4.7
